libraries/botbuilder-core/src/botAdapter.ts

```diff
@@ -97,6 +101,24 @@ export abstract class BotAdapter {
         logic: (revocableContext: TurnContext) => Promise<void>
     ): Promise<void>;
 
+    /**
```

Comment:
Try to reconcile signatures with .NET https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder/CloudAdapterBase.cs#L167